aruna/api/storage/services/v1/collection_service.proto

```diff
@@ -87,7 +87,7 @@ service CollectionService {
   rpc PinCollectionVersion(PinCollectionVersionRequest)
       returns (PinCollectionVersionResponse) {
     option (google.api.http) = {
-      post : "/v1/collection/{collection_id}/pin"
+      post : "/v1/workspace/"
```

Comment:
I don't think this change ended up here on purpose.
We should roll that back since this endpoint is actually allocated for creating a workspace. 

```suggestion
      post : "/v1/collection/{collection_id}/pin"
```